autopts/ptsprojects/zephyr/cap.py

```diff
@@ -173,12 +173,17 @@ def set_addr(addr):
     ]
 
     general_conditions = [
-        TestFunc(announcements, adv_data, rsp_data, False),
+        TestFunc(announcements, adv_data, rsp_data, False, SINK_CONTEXTS, SOURCE_CONTEXTS),
         TestFunc(btp.gap_adv_ind_on, ad=adv_data, sd=rsp_data),
     ]
 
     targeted_conditions = [
-        TestFunc(announcements, adv_data, rsp_data, True),
+        TestFunc(announcements, adv_data, rsp_data, True, SINK_CONTEXTS, SOURCE_CONTEXTS),
+        TestFunc(btp.gap_adv_ind_on, ad=adv_data, sd=rsp_data),
+    ]
+
+    idle_targeted_conditions = [
+        TestFunc(announcements, adv_data, rsp_data, True, 0, 0),
```

Comment:
```suggestion
        TestFunc(announcements, adv_data, rsp_data, True, Context.PROHIBITED, Context.PROHIBITED),
```

The fact that `0 == Context.PROHIBITED` makes the new requirement from CAP (1.0.1) even worse...

Reply:
I deliberately refrained from using the define of zero here because that would look worse... :P

Reply:
:D 

Ideally we'd use some defined value here, but I'm not happy about introducing a `Context.NONE` either, as that is not a BT SIG defined value... 

In hindsight, they should have defined these to use `NONE = 0xFFFF` or something, but alas...